packages/fxa-content-server/app/tests/spec/views/connect_another_device.js

```diff
@@ -385,14 +390,14 @@ describe('views/connect_another_device', () => {
 
       it('shows the marketing area, logs appropriately', () => {
         assert.isTrue(view._isSignedIn.called);
-        assert.lengthOf(view.$('.marketing-area'), 1);
+        assert.lengthOf(view.$('.pair-everywhere-cta'), 2);
         testIsFlowEventLogged('signedin.true');
         testIsFlowEventLogged('signin.ineligible');
         testIsFlowEventLogged('install_from.fx_desktop');
       });
 
       it('shows the success message', () => {
-        assert.lengthOf(view.$('.success'), 1);
+        assert.lengthOf(view.$('#fxa-connected-heading'), 1);
```

Comment:
Q: Does it make sense to import the ../../../functional/lib/selectors.js file and use the newly added `selectors.CONNECT_ANOTHER_DEVICE.CONNECTED` selector? Or is that crossing too many streams?

Reply:
I guess we could also just make it a local file/suite variable too, since we use it 5&times; in this file.
Mildly DRYer and doesn't involve pulling selectors from the functional test suite.

Or not, just something I saw. Feel free to whimsically disregard since it's a nit at best.